core/js/typespecs/common_types.ts

```diff
@@ -13,7 +15,21 @@ export const jsonSchema: z.ZodType<Json> = z.lazy(() =>
   z.union([literalSchema, z.array(jsonSchema), z.record(jsonSchema)]),
 );
 
-export const datetimeStringSchema = z.string().datetime({ offset: true });
+export const datetimeStringSchema = z
```

Comment:
This solves https://linear.app/braintrustdata/issue/BRA-1399/different-timestamps-returned-by-create-delete-patch-and-list-group by having zod transform the string to a datetime.